daisy/disk.go

```diff
@@ -212,17 +213,17 @@ func (dr *diskRegistry) detachHelper(dName, iName string, isAttached bool, s *St
 }
 
 // registerAttachment is called by Instance.regCreate and AttachDisks.validate and marks a disk as attached to an instance by Step s.
-func (dr *diskRegistry) regAttach(dName, iName, mode string, s *Step) DError {
+func (dr *diskRegistry) regAttach(deviceName, diskName, iName, mode string, s *Step) DError {
 	dr.mx.Lock()
 	defer dr.mx.Unlock()
 
-	pre := fmt.Sprintf("step %q cannot attach disk %q to instance %q", s.name, dName, iName)
+	pre := fmt.Sprintf("step %q cannot attach disk %q to instance %q", s.name, deviceName, iName)
```

Comment:
The message is not in sync with the one above which uses "disk with device name %q". Error messages should be consistent.

Reply:
Can't 100% sync with above actually, since detach is based on 'device name' while attach is based on 'disk name'. I should put 'diskName' here.